<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QScanModeView</class>
 <widget class="QWidget" name="QScanModeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QScanModeView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 16px &quot;Microsoft YaHei&quot;;
	color: #B4C4D9;
	background-color: rgba(0,0,0,0)
}
QLabel{
	background-color: rgba(0,0,0,0);

}
QSlider{
		background-color: rgba(0,0,0,0)
}


QCheckBox
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
	background-color: rgba(0,0,0,0)
}
QCheckBox::indicator
{
width: 20px;
height: 16px;
}
QCheckBox::indicator:unchecked
{
image: url(:/CArmWorkStation/ImageSource/Patient/uncheck.png);
}

QCheckBox::indicator:checked
{
image: url(:/CArmWorkStation/ImageSource/Patient/check.png);
}

QRadioButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
spacing:10;
background-color: rgba(0,0,0,0)
}
  QRadioButton::indicator {
      width: 16px;
      height: 16px;
  }

  QRadioButton::indicator::unchecked {
      image: url(:/SysConfig/ImageSource/SysConfig/RadioButtom_uncheck.png);
  }
/*
  QRadioButton::indicator:unchecked:hover {
      image: url(:/images/radiobutton_unchecked_hover.png);
  }

  QRadioButton::indicator:unchecked:pressed {
      image: url(:/images/radiobutton_unchecked_pressed.png);
  }
*/
  QRadioButton::indicator::checked {
      image: url(:/SysConfig/ImageSource/SysConfig/RadioButtom_check.png);
  }
/*
  QRadioButton::indicator:checked:hover {
      image: url(:/images/radiobutton_checked_hover.png);
  }

  QRadioButton::indicator:checked:pressed {
      image: url(:/images/radiobutton_checked_pressed.png);
  }
*/</string>
  </property>
  <widget class="QLabel" name="label_100">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>244</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>kVp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Scan_mA">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>287</y>
     <width>67</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	background:rgba(118,133,154,0.2);
	border:1px solid rgba(74,82,90,1);

}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Scan_pulseSelect">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>62</y>
     <width>274</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>274</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>34</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
border: 1px solid rgba(74,82,90,1);
border-radius:2px;
background:rgba(118,133,154,0.2);
color:#B4C4D9;
font:normal 14px  Microsoft YaHei;
}
QComboBox QAbstractItemView::item
{
height: 34px;
}
QComboBox:focus
{
border:3px solid #9dadc1;
border-radius:6px
}
QComboBox::down-arrow
{
image: url(:/CArmWorkStation/ImageSource/Patient/Doc_list_btn_pulldown_nor.png);
height:30px;
width:30px;
}
QComboBox:drop-down
{
subcontrol-origin: padding; 
subcontrol-position: top right;
min-width: 20px;border:none;
}
QListView::item
{
padding-left:30px;
background:0000ff;
color:#ffffff;
font:normal 16px Microsoft YaHei;
}
QComboBox:disabled
{
background:rgba(118, 133, 154, 0.05);
border: 1px solid rgba(74,82,90,0.5);
QSpinBox:disabled
{
background:rgba(118, 133, 154, 0.05);
border: 1px solid rgba(74,82,90,0.5);
color: rgba(180, 196, 217, 0.6);
}
}</string>
   </property>
   <item>
    <property name="text">
     <string>4PPS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8PPS</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>170</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{

		background-color: rgba(0,0,0,0);
}</string>
   </property>
   <widget class="QRadioButton" name="Scan_handWork">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>0</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>手动</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Scan_VA_Auto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>自动</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="Scan_mASlider">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>284</y>
     <width>400</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{height: 4px;background-color:rgba(66,80,96,1) ;border: 1px solid #566576;border-radius: 0px;padding - left:-1px;padding right:-1px;}
        QSlider::sub-page:horizontal{height: 4px;background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);background: qlineargradient(x1 : 0, y1 : 0.2, x2 : 1, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
    border: 1px solid #4A708B;
        border - radius: 2px;
    }
      QSlider::handle:horizontal
    {
    width: 22px;
        background-color:rgba(255,255,255,1);
margin-top: -10px;
margin-bottom: -40px;
    }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="Scan_kVp">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>232</y>
     <width>67</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	background:rgba(118,133,154,0.2);
	border:1px solid rgba(74,82,90,1);

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_154">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>171</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font:14px;

}</string>
   </property>
   <property name="text">
    <string>标准电压电流</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Scan_defaultMode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>22</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>设定为默认模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_93">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>72</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	font:14px;
}</string>
   </property>
   <property name="text">
    <string>默认脉冲选择</string>
   </property>
  </widget>
  <widget class="QSlider" name="Scan_kVpSlider">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>228</y>
     <width>400</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{height: 4px;background-color:rgba(66,80,96,1) ;border: 1px solid #566576;border-radius: 0px;padding - left:-1px;padding right:-1px;}
        QSlider::sub-page:horizontal{height: 4px;background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);background: qlineargradient(x1 : 0, y1 : 0.2, x2 : 1, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
    border: 1px solid #4A708B;
        border - radius: 2px;
    }
      QSlider::handle:horizontal
    {
    width: 22px;
        background-color:rgba(255,255,255,1);
margin-top: -10px;
margin-bottom: -40px;
    }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_101">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>298</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>mA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_94">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>126</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	font:14px;
}</string>
   </property>
   <property name="text">
    <string>默认剂量</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Scan_defaultDosimeter">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>116</y>
     <width>274</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>274</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>274</width>
     <height>34</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
border: 1px solid rgba(74,82,90,1);
border-radius:2px;
background:rgba(118,133,154,0.2);
color:#B4C4D9;
font:normal 14px  Microsoft YaHei;
}
QComboBox QAbstractItemView::item
{
height: 34px;
}
QComboBox:focus
{
border:3px solid #9dadc1;
border-radius:6px
}
QComboBox::down-arrow
{
image: url(:/CArmWorkStation/ImageSource/Patient/Doc_list_btn_pulldown_nor.png);
height:30px;
width:30px;
}
QComboBox:drop-down
{
subcontrol-origin: padding; 
subcontrol-position: top right;
min-width: 20px;border:none;
}
QListView::item
{
padding-left:30px;
background:0000ff;
color:#ffffff;
font:normal 16px Microsoft YaHei;
}
QComboBox:disabled
{
background:rgba(118, 133, 154, 0.05);
border: 1px solid rgba(74,82,90,0.5);
QSpinBox:disabled
{
background:rgba(118, 133, 154, 0.05);
border: 1px solid rgba(74,82,90,0.5);
color: rgba(180, 196, 217, 0.6);
}
}</string>
   </property>
   <item>
    <property name="text">
     <string>低剂量</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>标准剂量</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高剂量</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
